specification/security/resource-manager/Microsoft.Security/stable/2024-01-31/examples/Integrations/DeleteIntegration_example.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "api-version": "2024-01-31",
```

Comment:
should be 2023-07-01-preview according to service

Reply:
changed